### ISSUE TITLE
Adding sky spectrum 

### ISSUE THREAD

**Issue by astro-blue** [sbailey/prospect#19]
It may be useful to add a sky spectrum so that one can check whether or not emission line features of objects are at the wavelengths of strong sky emission lines. This can give some hints about whether observed emission line features are due to residuals of sky lines.

**Comment by armengau** [comment]
@sbailey  : to my understanding, this is already covered by the noise display (from ivar)
I suppose that anyway ivar is derived at some point in the pipeline from sky spectra ?
Or do I miss something ?

**Comment by moustakas** [comment]
I agree that the `ivar` spectrum should be dominated by the `sky` spectrum.

**Comment by sbailey** [comment]
Formally, the ivar includes CCD read noise, object poisson noise, and cosmics, not just the noise from the sky spectrum, even if for most wavelengths of most targets it is dominated by the sky spectrum.

Since every additional thing to plot adds more data to the html download package and can impact GUI performance, I suggest that if we do add a sky spectrum, that it just be an average/typical sky spectrum across all of the spectra, and not a per-target unique sky spectrum.

**Comment by astro-blue** [comment]
Hi all, 

I understand that for most of the objects, the sky dominates the noise. 
My concern is that after the sky subtraction, there will be some residuals left in the targeted spectra. The amplitudes of the residuals may not be proportional to the sky flux and therefore the noise. Having a sky spectrum for comparison may help people to check whether or not the observed emission line feature is directly on top of a sky line.
